### DISCUSSION TITLE
Iterate training on trained models

### DISCUSSION THREAD

**Discussion by matbord** [ultralytics/yolov5#2669] [Q&A]
Hi,
I would like to train a model, save it and then use it to start another training. I want to do this because I don't have enough memory to fit all my dataset in one training.
I would like to know if this procedure is correct before starting.
if first model :
`!python train.py --img 1316 --batch 4 --epochs 20 --data ../path_to_data1 --cfg ./models/yolov5x.yaml --weights yolov5x.pt --name yolov5x_model1--cache`
else:
`!python train.py --img 1316 --batch 4 --epochs 20 --data../path_to_data2 --cfg ./models/yolov5x.yaml --weights yolov5x_model1.pt --name yolov5x_model2 --cache`

**Comment by glenn-jocher** [comment]
@matbord that should be fine, but FYI if you are simply loading a pretrained model and not modifying it's structure there is no need to pass a --cfg argument, a --weights argument will suffice:
```
python train.py --data mydata.yaml --weights yolov5s.pt
```
